app/Eloquent/Permission.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Permission extends Model
+{
+    protected $fillable = [
+    	'name',
+    ];
+
+    public function roles()
+    {
+    	return $this->belongsToMany(Role::class);
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/Permission.php</br>Spaces must be used to indent lines; tabs are not allowed